src/freenas-installer/etc/installer12.sh

```diff
@@ -1150,8 +1150,14 @@ menu_install()
         OS=TrueNAS
     fi
 
-    # Tell it to look in /.mount for the packages.
-    /usr/local/bin/freenas-install -P /.mount/${OS}/Packages -M /.mount/${OS}-MANIFEST /tmp/data
+    for i in $(jq -r '."iso"."'auto-install-packages'"."'default'" | join(" ")' /var/db/trueos-manifest.json)
+    do
+        echo "Installing package: ${i}"
+        pkg -c /tmp/data install -y ${i}
```

Comment:
Do we really need to install one pkg at a time? Cant we install all of them in one pass?
How the default installer does it?

Reply:
It uses a loop right now as well :P

Reply:
There should only be one port here ideally though - the metaport - so we can easily manage dependencies